internal/librariangen/release/release.go

```diff
@@ -244,12 +247,12 @@ func writeErrorResponse(dir string, err error) error {
 // source path for the production code, and another for the
 // generated snippets.
 //
-// The "main" module of google-cloud-go might be expected to use "."
-// as a source path, but that would end up getting changes from *all*
-// nested modules (which will expand over time). That module is likely
-// to have a snowflake-like configuration.
-func wholeRepoLibrary(lib *request.Library) bool {
-	return slices.Contains(lib.SourcePaths, ".")
+// The use of a special ID of "root-module" for the module of
+// google-cloud-go containing "civil", "rpcreplay" etc is slightly
+// hacky, but avoids creating special-purpose configuration which
+// is realistically only ever going to be used by a single module.
```

Comment:
I think maybe we need one more sentence explaining why this hack is the best choice vs the "special-purpose configuration". As it stands, I'm left wondering what that would look like and if it might be the better choice.

Reply:
I've erred on the side of verbosity in the comment...